bin/rubocop-daemon-wrapper

```diff
@@ -1,6 +1,18 @@
 #!/bin/bash
+
 set -e
 
+COMMAND_PREFIX=""
+
+if [ -n "$RUBOCOP_DAEMON_USE_BUNDLER" ]; then
+  COMMAND_PREFIX="bundle exec"
+fi
+
+if ! command -v rubocop-daemon; then
```

Comment:
```suggestion
if ! command -v rubocop-daemon > /dev/null; then
```

`command -v` prints the path of `rubocop-daemon` when `rubocop-daemon` is available, so we should redirect to `/dev/null` to get rid of this output.